protobufs/livekit_metrics.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/livekit";
+option csharp_namespace = "LiveKit.Proto";
+option ruby_package = "LiveKit::Proto";
+
+
+/*
+  Protocol used to record metrics for a specific session.
+
+  Clients send their timestamp in their own monotonically increasing time (e.g `performance.now` on JS).
+  These timestamps are then converted by the SFU to align with the session's timeline, relative to the session start time.
+  (at receive time)
+
+  A metric can be linked to a specific track by setting `track_sid`.
+*/
+
+
+// index starting at 4096 are for predefined labels (`MetricLabel`)
+enum MetricLabel {
+  AGENTS_LLM_TTFT = 0; // time to first token from LLM
+  AGENTS_STT_TTFT = 1; // time to final transcription
+  AGENTS_TTS_TTFB = 2; // time to first byte
```

Comment:
we should define `METRICS_LABEL_MAX_VALUE = 4096` or something like that. if we have it in the codegen it's available everywhere and we don't have to keep repeating it in comments.